htdocs/comm/action/class/actioncomm.class.php

```diff
@@ -1543,6 +1543,19 @@ public function build_exportfile($format, $type, $cachedelay, $filename, $filter
                     $event['uid']='dolibarragenda-'.$this->db->database_name.'-'.$obj->id."@".$_SERVER["SERVER_NAME"];
                     $event['type']=$type;
                     $datestart=$this->db->jdate($obj->datep)-(empty($conf->global->AGENDA_EXPORT_FIX_TZ)?0:($conf->global->AGENDA_EXPORT_FIX_TZ*3600));
+					
```

Comment:
This part of code seems useless. Because it set $dateend but $dateend is erase just after.
May be you forget to remove the line $dateend= ... after this new block ?

Reply:
ups, you are right